mode/src/processing/mode/android/SDKDownloader.java

```diff
@@ -192,13 +192,14 @@ protected Object doInBackground() throws Exception {
         
          // Normalize built-tools and platform folders to android-<API LEVEL>
         String actualName = platformsFolder.listFiles()[0].getName();
-        renameFolder(platformsFolder, "android-" + TARGET_SDK, actualName);
+        renameFolder(platformsFolder, "android-" + downloadUrls.platformVersion, actualName);
```

Comment:
Quick question. I notice you use downloadUrls to store information of downloaded components. I am considering to review this name and change it to a more descriptive name aligned to its content. What do you think @DeerajTheepshi?

Reply:
Yeah, A more specific name like packageHolder ? As its hold all information about packages? 
Also if we are changing this, downloadUrls belongs to class SDKUrlHolder, which can also be changed to packageManager or something similar?  @kfrajer